packages/admin-next/dashboard/src/routes/gift-cards/gift-card-detail/loader.ts

```diff
@@ -0,0 +1,21 @@
+import { AdminGiftCardsRes } from "@medusajs/medusa"
+import { Response } from "@medusajs/medusa-js"
+import { adminProductKeys } from "medusa-react"
+import { LoaderFunctionArgs } from "react-router-dom"
+
+import { medusa, queryClient } from "../../../lib/medusa"
+
+const giftCardDetailQuery = (id: string) => ({
+  queryKey: adminProductKeys.detail(id),
+  queryFn: async () => medusa.admin.giftCards.retrieve(id),
+})
+
+export const giftCardLoader = async ({ params }: LoaderFunctionArgs) => {
```

Comment:
almost looks _remixy_

Reply:
When the `unstable_` is gone here I am pulling the trigger 🔫 

https://remix.run/docs/en/main/future/vite